src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -143,6 +152,18 @@ public boolean publish(String message, String color) {
     @Override
     public boolean publish(String message, JSONArray attachments, String color) {
         boolean result = true;
+        CloseableHttpClient client = getHttpClient();
```

Comment:
We don't close the client? :thinking:

Reply:
No, it wasn't closed before either. I just moved it to earlier in this method. I can handle closing it.